draft-ietf-taps-interface.md

```diff
@@ -318,8 +321,8 @@ If none of the available transport protocols provides Preservation of Message
 Boundaries, but there is a transport protocol that provides a reliable ordered
 byte stream, an application could receive this byte stream as partial
 Messages and transform it into application-layer Messages.  Alternatively,
-an application might provide a Message Framer, which can transform a
-byte stream into a sequence of Messages ({{framing}}).
+an application might provide a Message Framer, which transforms a
```

Comment:
```suggestion
an application might provide a Message Framer, which can transform a
```
I suggest to keep the "can transform" because I don't think Message Framers *always* do this transformation.